debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlAntlrDdlParserTest.java

```diff
@@ -281,6 +281,24 @@ public void shouldParseFunctionNamedDatabase() {
         parser.parse(ddl, tables);
     }
 
+    @Test
+    @FixFor("DBZ-910")
+    public void shouldParseConstraintCheck() {
+        parser = new MysqlDdlParserWithSimpleTestListener(listener, true);
+
+        final String ddl =
+                "CREATE TABLE t1 (c1 INTEGER NOT NULL,c2 VARCHAR(22),CHECK (c2 IN ('A', 'B', 'C')));"
+              + "CREATE TABLE t2 (c1 INTEGER NOT NULL,c2 VARCHAR(22),CONSTRAINT c1 CHECK (c2 IN ('A', 'B', 'C')));"
```

Comment:
There should be a test for `CREATE TABLE` with `CONSTRAINT` but without an identifier (as you've done it already for `ALTER TABLE`).

Reply:
@gunnarmorling Added